types/node/test/buffer.ts

```diff
@@ -43,6 +45,13 @@ const result2 = Buffer.concat([utf8Buffer, base64Buffer] as readonly Uint8Array[
     const value4: number = kStringMaxLength;
 }
 
+// Module variables
+{
+    // $ExpectType number
+    bufferModule.INSPECT_MAX_BYTES;
+    bufferModule.INSPECT_MAX_BYTES = 0;
```

Comment:
A test that this errors for `Buffer.INSPECT_MAX_BYTES` might be desirable, but this is non-blocking.

Reply:
I don't think the "module-scoped" variable `INSPECT_MAX_BYTES` exists in `Buffer` interface? 

Also there are no similar tests for `kMaxLength` & `kStringMaxLength`, which lives in the same scope as `INSPECT_MAX_BYTES` does.

Reply:
That's my point: verifying that a statement like `Buffer.INSPECT_MAX_BYTES = n` expects a TS error, as I imagine this to be the most common mishap.

Reply:
Added the tests.